src/simulate/events/v1p1_MediaEvent_Paused.json

```diff
@@ -0,0 +1,69 @@
+{
+  "sensor": "https://example.edu/sensors/1",
+  "sendTime": "2017-10-01T21:28:59.000Z",
+  "dataVersion": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+  "data": [
+    {
+      "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+      "id": "urn:umich:engin:leccap:uuid:956b4a02-8de0-4991-b8c5-b6eebb6b4cab",
+      "type": "MediaEvent",
+      "actor": {
+        "id": "",
+        "type": "Person"
+      },
+      "action": "Paused",
+      "object": {
+        "id": "https://leccap.engin.umich.edu/leccap/viewer/r/woNYn8",
+        "type": "VideoObject",
+        "name": "Lecture recorded on 9/19/2017",
+        "dateCreated": "2017-10-01T16:08:02.000Z",
+        "dateModified": "2017-10-01T21:28:59.000Z",
+        "duration": "PT5099.000S",
+        "isPartOf": {
+          "id": "https://leccap.engin.umich.edu/leccap/site/ziiq5uzrd9k06e6vjew",
+          "type": "WebPage",
+          "name": "EECS 370 001 / 002 / 003 / 004",
+          "dateCreated": "2017-10-01T06:41:17.000Z"
+        }
+      },
+      "target": {
+        "id": "https://leccap.engin.umich.edu/leccap/viewer/r/woNYn8",
+        "type": "MediaLocation",
+        "currentTime": "PT4540.137S",
+        "isPartOf": {
+          "id": "https://leccap.engin.umich.edu/leccap/viewer/r/woNYn8",
+          "type": "VideoObject"
+        }
```

Comment:
`target.isPartOf` needs only the ID, like this:

```json
        "isPartOf": "https://leccap.engin.umich.edu/leccap/viewer/r/woNYn8"
```

I will update the model events in tl-its-umich-edu/udp to reflect this.

Reply:
Done